### ISSUE TITLE
Is there a sample for Signup/Signin with phone or email like built-in flow?

### ISSUE THREAD

**Issue by LuisDev99** [azure-ad-b2c/samples#383]
Hello, I am wondering if there's a sample that is identical to the built-in user flow for phone/email signup/signin:

![image](https://user-images.githubusercontent.com/30758539/160341187-cddf7d90-947d-4c38-8ebd-e9985098480e.png)

In the image above, there's an input that redirects you to phone signin or email signin depending on whether you typed a phone or email. There's also separate links for signing up with phone or email and they take you to the corresponding signup screens. Is there something like this already? 

I am aware that there's this [starter pack](https://github.com/Azure-Samples/active-directory-b2c-custom-policy-starterpack/tree/main/scenarios/phone-number-passwordless) but it doesn't work (refer to this issue: https://github.com/Azure-Samples/active-directory-b2c-custom-policy-starterpack/issues/119 , but basically signup is broken and I cannot know why and also that starter is outdated) so I'm wondering if there's something like that in here that actually works 🥇 It would be huge, Thanks

**Comment by LuisDev99** [comment]
I was able to solve this issue by using the deployment tool: https://aka.ms/iefsetup (God bless the guy who developed this)

Basically, in that deployment tool, there's a checkbox that uploads policies for phone or email SUSI: 

![image](https://user-images.githubusercontent.com/30758539/160755848-b770cf0a-037e-48b5-944f-8fdf82230260.png)

Once the deployment tool is done, you'll see that it uploads the files from the [starter pack](https://github.com/Azure-Samples/active-directory-b2c-custom-policy-starterpack/tree/main/scenarios/phone-number-passwordless): 

![image](https://user-images.githubusercontent.com/30758539/160756048-9324aa2a-c292-46db-a5a3-940f043c7611.png)


In conclusion, I was manually modifying the custom policies from the starter pack (not good) but with the deployment tool, it uploaded some prebuilt policies that actually works!
